packages/lesswrong/components/common/FlashMessages.tsx

```diff
@@ -15,6 +15,39 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 });
 
+export const MessageContextProvider = ({children}: {
+  children: ReactNode
+}) => {
+  const [messages,setMessages] = useState<AnyBecauseTodo[]>([]);
```

Comment:
I was going to ask about the types here but then I saw that this was just existing code that got refactored